libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -54,9 +54,14 @@ export class RecoveryPhoneService {
    * by sending the phone number provided an OTP code to verify.
    * @param uid The account id
    * @param phoneNumber The phone number to register
+   * @param localizedMessageBody Optional localized message body
    * @returns True if code was sent and stored
    */
-  public async setupPhoneNumber(uid: string, phoneNumber: string) {
+  public async setupPhoneNumber(
+    uid: string,
+    phoneNumber: string,
+    localizedMessageBody?: { part1: string; part2: string }
```

Comment:
Instead of doing part1 and part 2, another approach is a template function, e.g.

`localizedMessageBody?: (code:string) => string`

This is a bit cleaner, because it abstracts away any assumptions about the number of parts in the message.

Reply:
Yes, I see what you mean! I'll have to modify our localization handling for auth-server a bit because it wasn't set up to take in variables, but should be a simple enough change to make this work.